### DISCUSSION TITLE
Markdown checkbox is not valid in vitepress ??

### DISCUSSION THREAD

**Discussion by xxxxue** [vuejs/vitepress#2784] [Q&A]
![image](https://github.com/vuejs/vitepress/assets/32764266/dab7668e-96b1-4717-acdd-54c2116af288)

**Comment by brc-dd** [comment]
Checkbox is not part of the markdown standard. Refer: https://github.com/vuejs/vitepress/issues/413#issuecomment-1017488394 or https://github.com/vuejs/vitepress/issues/1923#issuecomment-1431479500
